src/RoomsSDKAdapter.integration.test.js

```diff
@@ -0,0 +1,47 @@
+import {removeTestUser} from '@webex/test-users';
+
+import {createIntegrationTestUser, createIntegrationSdkInstance} from './testHelper';
+import RoomsSDKAdapter from './RoomsSDKAdapter';
+
+describe('Rooms SDK Adapter Interface', () => {
+  let roomsSDKAdapter;
+  let sdk;
+  let user;
+  let createdRoom;
+
+  beforeAll(async () => {
+    // Setting timeout higher for user creation and room creation
+    jest.setTimeout(15000);
+
+    user = await createIntegrationTestUser();
+
+    sdk = createIntegrationSdkInstance(user.token);
+
+    createdRoom = await sdk.rooms.create({title: 'Webex Teams Test Room'});
+  });
+
+  afterAll(() =>
```

Comment:
Can we move this to the end of the block? e.g.

```
beforeAll()
beforeEach()

test

afterEach()
afterAll()

Reply:
The only thing i found was [this jest plugin](https://www.npmjs.com/package/eslint-plugin-jest) but its rules say that they all should be on the top of the describe

Reply:
I actually prefer the grouping that way. All of the setup and teardown hooks are supposed to be before the tests. All of the jest examples (https://jestjs.io/docs/en/setup-teardown) show it this way. 

https://github.com/jest-community/eslint-plugin-jest/blob/HEAD/docs/rules/prefer-hooks-on-top.md

Reply:
Ok, we can open a different PR to add the jest linter